cl/recap/tests.py

```diff
@@ -4498,3 +4498,88 @@ def test_retry_webhooks_integration(
                     self.assertEqual(
                         webhook_triggered[0].next_retry_date, seven_retry_time
                     )
+
+    def test_webhook_disabling(
+        self,
+        mock_bucket_open,
+        mock_cookies,
+        mock_pacer_court_accessible,
+        mock_get_document_number_from_confirmation_page,
+    ):
+        """This test checks if we can send properly failing webhook events
+        notifications and disable the webhook after a webhook event is max retried.
+        """
```

Comment:
These are usually done as a question, so:

```suggestion
        """Can we properly send failing webhook events and disable webhooks 
        if max retries are expired?
        """
```

Reply:
Changed.